.github/workflows/test-build-deb.yml

```diff
@@ -593,7 +593,7 @@ jobs:
         with:
           repository: opendistro-for-elasticsearch/kibana-oss
           ref: ${{env.es_version}}
-          token: ${{ secrets.ODFE_RELEASE_BOT_PRIVATE_READ_WRITE_TOKEN }}
+          token: ${{ secrets.ODFE_RELEASE_BOT_PUBLIC_PRIVATE_READ_WRITE_TOKEN }}
```

Comment:
Why did we change the token again?

Reply:
@gaiksaya Apparently after testing the private only token does not work at all, and is no need to create a separate token just for a reading purpose. I just use the same token for all now. If we have access to private there is no reason it cannot be used for public. For public read only there is another token for that purpose.